coursera/api.py

```diff
@@ -543,8 +544,15 @@ def extract_links_from_lecture(self,
         if assets is None:
             assets = []
 
-        links = self._extract_videos_and_subtitles_from_lecture(
-            video_id, subtitle_language, resolution)
+        try:
+            links = self._extract_videos_and_subtitles_from_lecture(
+                video_id, subtitle_language, resolution)
+        except HTTPError as ex:
+            FORBIDDEN = 403
+            if ex.response.status_code == FORBIDDEN:
```

Comment:
I don't think we should only catch 403 error. Catching whole class of HTTPError is better to me.